README.md

```diff
@@ -1,3 +1,8 @@
+> [!WARNING]
+> As of the 1.22.3 release, this repo is no longer being used to
+> release leveldb for CocoaPods. Instead, refer to
+> https://github.com/firebase/leveldb/tree/firebase-release.
```

Comment:
I was going to rename the `spm` branch to something more general since the pod now lives in it. WDYT about `firebase-release`? Any preferred alternatives?

Reply:
SGTM! Seeing this after I sent a similar email to you. 😄